core/src/Kokkos_SYCL.hpp

```diff
@@ -187,6 +187,41 @@ struct DeviceTypeTraits<Kokkos::Experimental::SYCL> {
 }  // namespace Experimental
 }  // namespace Tools
 
+namespace Experimental {
+template <class... Args>
+std::vector<SYCL> partition_space(const SYCL& sycl_space, Args...) {
+#ifdef __cpp_fold_expressions
```

Comment:
What is the point of this?  We require C++17 for the SYCL backend?

Reply:
Sure, we could remove it.